README.md

```diff
@@ -34,3 +34,106 @@ Please see the [Juju SDK docs](https://juju.is/docs/sdk) for guidelines
 on enhancements to this charm following best practice guidelines, and
 [CONTRIBUTING.md](https://github.com/charmed-kubernetes/charm-azure-cloud-provider/blob/main/CONTRIBUTING.md)
 for developer guidance.
+
+
+## Installation
+
+Deploying the charm into an existing juju machine model with `charmed-kubernetes` is as simple as 
+
+```bash
+$ juju deploy kubevirt --channel <channel> --config operator-release='<v0.min.bug>' --trust
```

Comment:
no $ nonsense!
https://docs.ubuntu.com/styleguide/en/#code-examples-in-documentation